tf/variables.tf

```diff
@@ -1,37 +1,35 @@
 variable "client_id" {}
 variable "client_secret" {}
+variable "ssh_public_key" {
```

Comment:
Having this empty means it will consume an environment variable or one specified by command line.  We inject the public key via our secrets pipeline.  The private key is also injected in this way.  I don't like moving keys around like this - we need to do something better long term I think.